docs/doxygen/overviews/xrc_format.h

```diff
@@ -384,6 +384,42 @@ If both specifications are provided, then @c stock_id is used if it is
 recognized by wxArtProvider and the provided bitmap file is used as a fallback.
 
 
+@subsection overview_xrcformat_type_bitmaps BitmapBundle
```

Comment:
I think it would be nice to call it something hinting as its purpose, e.g. "Multi-resolution bitmap" perhaps?